test/Common.h

```diff
@@ -24,6 +24,17 @@
 #include <boost/program_options.hpp>
 #include <boost/noncopyable.hpp>
 
+#ifdef _WIN32
+#define EVMONE_FILENAME "evmone.dll"
+#define EVMONE_DOWNLOADLINK "https://github.com/ethereum/evmone/releases/download/v0.1.0/evmone-0.1.0-windows-amd64.zip"
+#elif defined(__APPLE__)
+#define EVMONE_FILENAME "libevmone.dylib"
+#define EVMONE_DOWNLOADLINK "https://github.com/ethereum/evmone/releases/download/v0.1.0/evmone-0.1.0-darwin-x86_64.tar.gz"
+#else
+#define EVMONE_FILENAME "libevmone.so"
+#define EVMONE_DOWNLOADLINK "https://github.com/ethereum/evmone/releases/download/v0.1.0/evmone-0.1.0-linux-x86_64.tar.gz"
+#endif
+
```

Comment:
Maybe we should talk about this in general somewhat soon, but I'd I'd highly recommend avoiding macros but instead use `constexpr` instead (so here: `constexpr std::string_view ...;`

Reply:
Yes, constexprs would be better - if that is possible.

Reply:
I'd only use macros, for platform specific distinctions like this, where you need the preprocessor for the distinction anyways... but one can very well argue that even then one could/should use a constexpr instead... I'm wondering if the compiler compile-time-concatenates things like "String prefix" STRING_MACRO "string suffix", if rewritten with constant expressions (it might very well not, but well, it's probably nothing we should really care about).

Reply:
I actually think this will make things uglier due to the need of manual string concatenation... but ok, I'll change it.

Reply:
Did it in the last commit noiw, but personally I actually prefer the version with macros in this particular case.

Reply:
Note that ``boost::filesystem::path::operator/`` has no ``string_view`` overloads - and that boost program options doesn't specify, if the description, which is passed as char array, may or may not be a temporary, both of which cause minor complications with ``constexpr`` as seen in the respective commit.